### PR TITLE
fix(webhook-slash): remove comment about root path

### PR DESCRIPTION
**Pull request by aetheryx**
Discord has added support for non-root paths

### CODE REVIEW

**Review by Erk- — APPROVED** [review]

**Comment by vilgotf** [comment]
Shouldn't this check be removed all together?

**Comment by Erk-** [comment]
Not nessesarily, this is just an example so it in my opinion also shows that you could filter by other subdomains if you want to

**Comment by 7596ff** [comment]
> you could filter by other subdomains if you want to

Can we add a note about this as well?

File: webhook-slash/src/main.rs [hunk 1]
```diff
@@ -38,8 +38,8 @@ where
             .status(StatusCode::METHOD_NOT_ALLOWED)
             .body(Body::empty())?);
     }
-    // Check if the path the request is sent to is the root of the domain,
-    // this is the only path supported by Discord.
+
+    // Check if the path the request is sent to is the root of the domain.
```
Comment (7596ff) [inline webhook-slash/src/main.rs:42]:
```suggestion
    // Check if the path the request is sent to is the root of the domain.
    //
    // This filter is for the purposes of this example. The user may filter by
    // any subdomain they choose.
```

**Review by 7596ff — CHANGES_REQUESTED** [review]

**Review by Erk- — APPROVED** [review]

**Review by 7596ff — APPROVED** [review]

**Pull request merged**